### DISCUSSION TITLE
Example of slash commands - more examples for newbies

### DISCUSSION THREAD

**Discussion by iagolirapasssos** [Rapptz/discord.py#7847] [Q&A]
The current form of the slash commands is different from the language used in the previous dpy... I tested some examples, like [exemples](https://gist.github.com/AbstractUmbra/a9c188797ae194e592efe05fa129c57f#file-extension_with_group-py-L13), but I couldn't get the bot to come online. 

![cap000000](https://user-images.githubusercontent.com/4522963/161824418-843de93e-2d4a-4f3d-97f5-aab1bec4932c.png)
I noticed that when running the examples the bot is not in a loop.
![cap000000](https://user-images.githubusercontent.com/4522963/161824314-f8e2eb1f-20de-4009-8395-dfd52e19b2f4.png)

**Comment by jontobonto** [comment]
Did you sync the commands?

**Reply by iagolirapasssos** [reply]
yeah

**Reply by iagolirapasssos** [reply]
Can you send a simple example with cogs?

**Reply by LeoCx1000** [reply]
You can find one in [this gist](https://gist.github.com/AbstractUmbra/a9c188797ae194e592efe05fa129c57f)

**Reply by iagolirapasssos** [reply]
thanks
